src/x11.cc

```diff
@@ -33,6 +33,9 @@
 #include "x11.h"
 
 #include <X11/X.h>
+#include <X11/Xlibint.h>
+#undef min
+#undef max
```

Comment:
Required on Void Linux - Author stated in related issue (#1939) the code doesn't compile without this include.